src/amo/components/Review/index.js

```diff
@@ -37,33 +37,38 @@ type AddonReviewProps = {|
   updateReviewText: (review: $Shape<SubmitReviewParams>) => Promise<void>,
 |};
 
-type AddonReviewState = {|
+type ReviewState = {|
   reviewBody: ?string,
 |};
 
-export class AddonReviewBase extends React.Component {
+// Changing this will invalidate any saved add-ons for users because it
+// changes the prefix we use in our offline storage key.
+const LOCALSTATE_PREFIX = 'AddonReview';
```

Comment:
Added this because I realised any future over-zealous search+replaces might not catch this and it would render saved reviews un-get-able.